### ISSUE TITLE
Assignment 9

### ISSUE THREAD

**Issue by jisaacks** [tiy-tpa-js-q1-2015/assignments#91]
https://github.com/tiy-tpa-js-q1-2015/assignments/tree/master/009#021015

**Comment by tawnimartin** [comment]
Hi JD!

Here's my homework:
https://github.com/tawnimartin/tiy-assignment-9/tree/gh-pages
http://tawnimartin.github.io/tiy-assignment-9/

The calculator works, except for using the decimal point.  It's because I used parseFloat(".") when I added it to the variable (all by itself) which essentially ignores it. This could definitely be done better!  :-(  I could have used arrays rather than variables and parseFloated the final number in the calc function, which would have preserved the decimal point. I also have a nested if statement in there.  Ah well, this is where it's at for now!

Tawni
